rotkehlchen/tests/exchanges/test_coinbase.py

```diff
@@ -652,6 +660,33 @@ def test_coinbase_query_trade_history_unexpected_data(function_scope_coinbase):
     )
 
 
+def test_coinbase_query_trade_history_paginated(function_scope_coinbase):
+    """Test that coinbase trade history query handles unexpected data properly"""
```

Comment:
Can you adjust the docstring to describe the test properly? I guess this is just a copy paste from another test.